examples/transmitter/swagger_server/events.py

```diff
@@ -56,5 +87,9 @@ class SecurityEvent(BaseModel):
 
 SUPPORTED_EVENTS = [
     VerificationEvent.__uri__,
-    SessionRevoked.__uri__
+    CAEPEvent.__uri__+"session-revoked",
```

Comment:
Instead of creating the URIs again here, please refer to the ones in the event models, for example: `SessionRevoked.__uri__`